CMSgov/design-system
# [WNMGDS-1432] Add error icon to html error examples

## Summary

HTML examples of error messages didn't all use the SVG error Icon this PR updates the HTML examples to use the error icon. 

### Added

- Added error SVG to HTML examples where it was missing 

## How to test

- Run the doc site locally `yarn start` and see that error icons are showing for HTML examples for the following: Radio/checkbox, form label, dropdown, date field, text field
